icu4c/source/i18n/unitconverter.h

```diff
@@ -7,16 +7,26 @@
 #ifndef __UNITCONVERTER_H__
 #define __UNITCONVERTER_H__
 
-
-#include <vector>
+#include "cmemory.h"
+#include "unicode/errorcode.h"
+#include "unicode/measunit.h"
+#include "unitconverter.h"
+#include "unitsdata.h"
 
 U_NAMESPACE_BEGIN
 
+enum U_I18N_API UnitsConvertibilityState {
+    RECIPROCAL,
+    CONVERTIBLE,
+    UNCONVERTIBLE,
+};
 
+UnitsConvertibilityState U_I18N_API
+checkConvertibility(const MeasureUnit &source, const MeasureUnit &target,
+                const MaybeStackVector<ConversionRateInfo> &conversionRateInfo, UErrorCode &status);
```

Comment:
formatting nit: indentation? (clang-format should indent this to align?)

Reply:
let's defer this after we submit .clang-format file :)

Reply:
it is not fixed 👍